reproman/tests/test_cmdline_main.py

```diff
@@ -77,6 +77,7 @@ def test_help_np():
     # but order is still not guaranteed (dict somewhere)! TODO
     # see https://travis-ci.org/reproman/reproman/jobs/80519004
     # thus testing sets
+    # TODO(asmacdo) On fedora "*Global Options*" renders as "Options" and is not included in sections
```

Comment:
sounds like a python-version related thing we also fixed for some other case , rests in ad-hoc tuning of argparser setup -- are you sure this is still the case?

Reply:
nope- you fixed this one :)